File: Python 3/Phyton Curso em Video/Mundo1/Exercicios/Exerc/ex008.py
#  ESCREVA UM PROGRAMA QUE LEIA UM VALOR EM METROS E O EXIBA CONVERTIDO EM CENTÍMETROS E MILÍMETROS
# EXTRA - KM, HECTOMETROS, DECAMETROS, DECÍMETROS E MILÍMITROS

m = float(input('Digite quantos metros você quer converter: '))
km = m / 1000
hec = m / 100
deca = m / 10
dcm = m * 10
cm = m * 100
mm = m * 100
print('Kilômetros {} km'.format(km))
print('Hectômetro: {} hm'.format(hec))
print('Decâmetro: {:.0f} decam'. format(deca))
print('Decímetro: {:.0f} dcm'.format(dcm))
print('Centímetro: {:.0f} cm'.format(cm))
print('Milímetro: {:.0f} mm'.format(mm))

File: JavaScript/exercícios/ex008/ex008.js
// Escreva um programa que leia um valor em metros e o exiba convertido em centímetros e milímetros Extra - km, hectometros, decametros, decimetros

function conversor(){
    let n1 = document.getElementById('txtn1')
    let res = document.getElementById('res')
    let n = Number(n1.value)
    res.innerHTML = `Quilômetros = ${n / 1000} km<br>Hectômetros = ${n / 100} hm<br>Decâmetros = ${n / 10} dam<br>Decímetros = ${n * 10} dm<br>Centímetros = ${n * 100} cm<br>Mílimetros = ${n * 1000} mm`
}